### PR TITLE
Get inputs instead of msg/signature

### PR DESCRIPTION
**Pull request by smyzin**
- got `inputs` instead of msg + signature
- removed `generateInput` step
- added new model

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +26,15 @@
     "@ethereumjs/util": "^8.0.2",
     "@hapi/boom": "^10.0.0",
     "@koa/cors": "^4.0.0",
+    "@types/json-bigint": "^1.0.1",
```
Comment (T-Damer) [inline package.json]: Should be in `devDeps`

File: src/controllers/prove.ts [hunk 2]
```diff
@@ -1,16 +1,24 @@
+import * as JSONbig from 'json-bigint'
```
Comment (T-Damer) [inline src/controllers/prove.ts]: Can we import only `parse`?

File: src/controllers/prove.ts [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
 import ProofResultParams from '@/validators/ProofResultParam'
 
 @Controller('/')
 export default class ProveController {
   @Post('/')
-  async proof(@Body({ required: true }) input: Inputs) {
-    const job = await JobModel.create({ input })
+  async proof(@Body({ required: true }) input: JsonProofInput) {
+    const { TPreComputes, U, s } = input
+    const parsedInput: ProofInput = {
```
Comment (T-Damer) [inline src/controllers/prove.ts:16]: Do you need to explicitly specify a type here? I think the `parse` should return a proper type, if it doesn't, than this is fine

**Review by T-Damer — APPROVED** [review]

File: src/controllers/prove.ts (hunk 3 above)
Comment (smyzin) [reply src/controllers/prove.ts:16]: Without the `ProofInput` type, the const will have any inside object: 
```
{
  TPreComputes: any
  U: any
  s: any
}
```

**Pull request merged**